crates/wasmtime/src/runtime/component/concurrent.rs

```diff
@@ -4006,7 +4486,7 @@ pub struct ConcurrentState {
     instance_states: HashMap<RuntimeComponentInstanceIndex, InstanceState>,
     /// The "high priority" work queue for this instance's event loop.
```

Comment:
```suggestion
    /// The "high priority" work queue for this store's event loop.
```

Not related to your changes, but I should have updated these docs when I moved `ConcurrentState` to `StoreOpaque`.